### DISCUSSION TITLE
Should we release 0.69.4?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#28] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/30

---

Current Release: [0.69.3](https://github.com/facebook/react-native/releases/tag/v0.69.3)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
we need to bump CLI to https://github.com/react-native-community/cli/releases/tag/v8.0.4 for this fix https://github.com/react-native-community/cli/pull/1655

**Reply by kelset** [reply]
PR: https://github.com/facebook/react-native/pull/34272

**Comment by adrianha** [comment]
hi, could we include changes from https://github.com/facebook/react-native/commit/87cfd386cb2e02bfa440c94706d9d0274f83070c to fix [issues](https://github.com/facebook/react-native/issues?q=is%3Aissue+is%3Aopen+DSO) related with soloader? I've seen this changes already included in 0.70 rc.

**Comment by matinzd** [comment]
Is there any way to release patch for these cli changes https://github.com/facebook/react-native/pull/34322 and https://github.com/facebook/react-native/pull/34275?
It fixes this issue as well https://github.com/facebook/react-native/issues/34247.

**Reply by kelset** [reply]
@matinzd I'm not sure I follow, you are linking CLI v9 which is the one paired with RN0.70. I just released 0.69.4 with a bump to 8.0.4 https://github.com/facebook/react-native/pull/34272 that maybe helps?

Anyway, if there are still commits that you want cherry-picked please ask over in #30
